### PR TITLE
Bump the screen coverage to 100%

### PR DESCRIPTION
**Pull request by VincentCATILLON**
## Detailed purpose of the PR

This PR bumps the screen coverage to 💯 %

### Testing strategy

- [x] I ensured my code is tested: unit tests
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/screens/correction.js [hunk 1]
```diff
@@ -230,14 +231,11 @@ const getAnswersState = createSelector(
...
   correction => {
-    const corrections = (correction && correction.corrections) || [];
-
-    return corrections.reduce((result, item) => {
-      if (item.answer) {
-        result.push(item.answer);
-      }
-      return result;
-    }, []);
+    const corrections = correction && correction.corrections;
```
Comment (VincentCATILLON) [inline src/screens/correction.js]: this line is incorrect

File: src/screens/correction.js [hunk 2]
```diff
@@ -230,14 +231,11 @@ const getAnswersState = createSelector(
...
   correction => {
-    const corrections = (correction && correction.corrections) || [];
-
-    return corrections.reduce((result, item) => {
-      if (item.answer) {
-        result.push(item.answer);
-      }
-      return result;
-    }, []);
+    const corrections = correction ? correction.corrections : [];
```
Comment (VincentCATILLON) [inline src/screens/correction.js]: the edge case is not covered :) should be done soon

**Comment by VincentCATILLON** [comment]
@Fausto95 no need at this time, almost nothing has been modified in components/screens as its only a tests coverage enhancement. the only one thing that has been fixed, is the error happening sometimes on correction screen due to incorrect type in `player-store`, fixed in this PR: https://github.com/CoorpAcademy/components/pull/1627

File: src/modules/environment.js [hunk 3]
```diff
@@ -10,14 +10,10 @@ export const __ENV__ = env.NODE_ENV;
 export const __TEST__ = __ENV__ === 'test';
 export const __DEV__ = __ENV__ === 'development';
 
-export const DEV_TOKEN =
```
[the remark this answers is not in the document]
Comment (VincentCATILLON) [reply src/modules/environment.js]: because it should be set in env file, this helper is only here to spot environments (tests, e2e, dev, production). exposing a token is not his responsability

File: src/redux/actions/clues.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// @flow strict
+
+import {getClue} from '@coorpacademy/player-store';
+
+export const fetchClue = () => getClue;
```
[the remark this answers is not in the document]
Comment (VincentCATILLON) [reply src/redux/actions/clues.js]: To avoid this: https://github.com/CoorpAcademy/mobile/pull/423/files#diff-fd82b19415ce10e5519e00c006151da4L89

Because having a custom action defined in screen should be testable, and so, this is not the role of the screen, it should be a separate action, that should be tested separately
